### PR TITLE
Updated polish translation in temperature@fevimu

### PR DESCRIPTION
**Pull request by WinLin97**

### CODE REVIEW

File: temperature@fevimu/files/temperature@fevimu/po/pl.po [hunk 1]
```diff
@@ -134,32 +134,40 @@ msgstr "Interwał odświeżania"
...
 
 #. settings-schema.json->change-color->tooltip
 msgid ""
 "Selecting this option, this applet becomes\n"
 " orange when the High Temperature Threshold is reached,\n"
 " red when the Critical Temperature Threshold is reached."
 msgstr ""
+„Zaznaczenie tej opcji spowoduje, że aplet zmieni kolor na:\n”
+„ pomarańczowy po osiągnięciu progu wysokiej temperatury,\n”
+„ czerwony po osiągnięciu progu krytycznej temperatury.”.
```
Comment (rcalixte) [inline temperature@fevimu/files/temperature@fevimu/po/pl.po:147]:
```suggestion
"Zaznaczenie tej opcji spowoduje, że aplet zmieni kolor na:\n"
" pomarańczowy po osiągnięciu progu wysokiej temperatury,\n"
" czerwony po osiągnięciu progu krytycznej temperatury.".
```

File: temperature@fevimu/files/temperature@fevimu/po/pl.po [hunk 2]
```diff
@@ -134,32 +134,40 @@ msgstr "Interwał odświeżania"
...
-msgstr ""
+msgstr "Używaj tylko kolorów, aby mnie ostrzec"
 
 #. settings-schema.json->only-colors->tooltip
 msgid ""
 "Select this option to not display text alerts in horizontal panels.\n"
 "(No text alert is ever displayed on vertical panels.)"
 msgstr ""
+„Zaznacz tę opcję, aby nie wyświetlać alertów tekstowych w panelach poziomych.\n”
+„(W panelach pionowych nigdy nie są wyświetlane alerty tekstowe).”
```
Comment (rcalixte) [inline temperature@fevimu/files/temperature@fevimu/po/pl.po:159]:
```suggestion
"Zaznacz tę opcję, aby nie wyświetlać alertów tekstowych w panelach poziomych.\n"
"(W panelach pionowych nigdy nie są wyświetlane alerty tekstowe)."
```

File: temperature@fevimu/files/temperature@fevimu/po/pl.po [hunk 3]
```diff
@@ -134,32 +134,40 @@ msgstr "Interwał odświeżania"
...
 
 #. settings-schema.json->keep_size->description
 msgid "Avoid size changes"
-msgstr ""
+msgstr "„Unikaj zmian rozmiaru”"
```
Comment (rcalixte) [inline temperature@fevimu/files/temperature@fevimu/po/pl.po:163]:
```suggestion
msgstr "Unikaj zmian rozmiaru"
```

File: temperature@fevimu/files/temperature@fevimu/po/pl.po [hunk 4]
```diff
@@ -134,32 +134,40 @@ msgstr "Interwał odświeżania"
...
 
 #. settings-schema.json->keep_size->tooltip
 msgid ""
 "Checked: The size of this applet remains constant, once all your preferences "
 "have been defined.\n"
 "Unchecked: The size of this applet is optimized, but it may change."
 msgstr ""
+„Zaznaczone: Rozmiar tego apletu pozostaje stały, gdy wszystkie preferencje ”
+„zostały zdefiniowane.\n”
+„Niezaznaczone: Rozmiar tego apletu jest zoptymalizowany, ale może ulec zmianie.”
```
Comment (rcalixte) [inline temperature@fevimu/files/temperature@fevimu/po/pl.po:173]:
```suggestion
"Zaznaczone: Rozmiar tego apletu pozostaje stały, gdy wszystkie preferencje "
"zostały zdefiniowane.\n"
"Niezaznaczone: Rozmiar tego apletu jest zoptymalizowany, ale może ulec zmianie."
```

**Review by rcalixte — CHANGES_REQUESTED** [review]
@WinLin97 You can avoid these errors by using something like Poedit to create the translations. Feel free to ask if you have any questions.

**Comment by WinLin97** [comment]
Sorry about the errors, I just wanted to add the missing strings and I do it from browser.

**Comment by rcalixte** [comment]
> Sorry about the errors, I just wanted to add the missing strings and I do it from browser.

No problem! Thank you for contributing!

**Review by rcalixte — APPROVED** [review]

**Pull request merged**